.github/workflows/unit-tests.yaml

```diff
@@ -28,8 +28,19 @@ jobs:
     steps:
     - name: Checkout msamp
       uses: actions/checkout@v2
+      with:
+          submodules: true
+    - name: Install nccl
+      run: |
+        cd third_party/nccl
+        make -j src.build NVCC_GENCODE="\
```

Comment:
Which docker image is used for unit test?

Reply:
nvcr: 22.09-py3, nvcr: 22.12-py3 and nvcr: 23.04-py3.

Reply:
For UT, we still use torch docker image from nvidia, not use ms-amp docker image. That's why we need to install nccl and set environment variable in UT pipeline.  That's reasonable because the MS-AMP docker image is always changing.

Reply:
I see. Thanks Yuxiang!